encrypt/encrypt.go

```diff
@@ -326,36 +326,21 @@ func calculateHashes(fileSet helpers.EncryptionFileSet) (*hashSet, error) {
 }
 
 // Reads a public key from a file using the crypt4gh keys module
-func readPublicKeyFile(filename string) (key *[32]byte, err error) {
```

Comment:
Doesn't it make more sense to keep this func name as is?
We still call keys.ReadPublicKey that does the actual pub key reading.

Reply:
but that call is still inside here... not sure really :)

Reply:
hmm good point. I originally kept that name for parity since `decrypt.go` has a respective `readPrivateKey` function.  But they should both be renamed since they both read from file whereas the respective functions from the `keys` library of `crypt4gh` read from stream. So, I renamed them.